### PR TITLE
fixed some typos

### PR DESCRIPTION
**Pull request by SyedAahana**

### CODE REVIEW

File: ThirdPartyNotices.txt [hunk 1]
```diff
@@ -698,7 +698,7 @@ Mozilla Public License, version 2.0
 
 2.4. Subsequent Licenses
 
-     No Contributor makes additional grants as a result of Your choice to
+     No Contributor makes additional grants as a result of your choice to
```
Comment (gjsjohnmurray) [inline ThirdPartyNotices.txt:701]: 'Your' is capitalized to denote it as a term defined in section 1 above.

File: ThirdPartyNotices.txt [hunk 2]
```diff
@@ -725,7 +725,7 @@ Mozilla Public License, version 2.0
 3.1. Distribution of Source Form
 
      All distribution of Covered Software in Source Code Form, including any
-     Modifications that You create or to which You contribute, must be under the
+     modifications that you create or to which you contribute, must be under the
```
Comment (gjsjohnmurray) [inline ThirdPartyNotices.txt:728]: Capitalizations denote terms defined in section 1.

File: ThirdPartyNotices.txt [hunk 3]
```diff
@@ -980,7 +980,7 @@ Copyright (c) textmate-make.tmbundle project authors
 
 If not otherwise specified (see below), files in this repository fall under the following license:
 
-Permission to copy, use, modify, sell and distribute this
+Permissions to copy, use, modify, sell and distribute this
```
Comment (gjsjohnmurray) [inline ThirdPartyNotices.txt:983]: 'Permission' was correct. We are being permitted to do several things.

**Comment by alexdima** [comment]
Thank you. The ThirdPartyNotices.txt file is generated based on metadata present in the repository and we shouldn't edit it directly. It looks like in the meantime `jeff-hykin/better-c-syntax` is credited correctly in TPN.txt.

**Pull request closed without merging**